cfme/tests/services/test_provision_stack.py

```diff
@@ -395,3 +395,39 @@ def test_stack_template_azure():
         upstream: yes
     """
     pass
+
+
+@pytest.mark.meta(automates=[1698439])
+@pytest.mark.meta(blockers=[BZ(1754543)])
+@pytest.mark.tier(2)
+@pytest.mark.provider([EC2Provider], selector=ONE, scope='module')
+def test_read_dialog_timeout_ec2_stack(order_stack):
+    """
+
+    Bugzilla:
+        1698439
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. create an aws template with an optional value "timeout"
+            2. create a dialog that will offer an option to overwrite "timeout"
+               with a custom value typed at input
+            3. Navigate to order page of service
+            4. provision using a non-zero value in timeout
+        expectedResults:
+            1.
+            2.
+            3.
+            4. the value input should be passed
+    """
+    msg = "<AEMethod groupsequencecheck> 	dialog_stack_timeout: 20"
```

Comment:
@niyazRedhat I think better to use regular expression for such match. I'm suspect about whitespace.

Reply:
Its okay here without it because there is no backslash, but for future use @niyazRedhat, its good to use the syntax for the string that includes an 'r' `r"<some-regex-string>"`.

This is a stringprefix in python, part of the string literal definition. It stands for `raw` string (not regex string) and will treat the backslashes regularly used in regex as literal.

https://docs.python.org/3.7/reference/lexical_analysis.html#grammar-token-stringprefix